inc/cachify.class.php

```diff
@@ -1290,7 +1302,8 @@ public static function set_cache( $data ) {
 			),
 			self::_cache_hash(),
 			self::_minify_cache( $data ),
-			self::_cache_expires()
+			self::_cache_expires(),
+			self::_signature_details()
```

Comment:
Why is it needed here? `Cachify_DB::store_item` does not expect this information.